tools/functional-tester/etcd-tester/cluster.go

```diff
@@ -48,12 +48,14 @@ type cluster struct {
 	stressKeySize        int
 	stressKeySuffixRange int
 
-	Size      int
-	Stressers []Stresser
+	Size           int
+	Stressers      []Stresser
+	LeaseStressers []Stresser
```

Comment:
Is is possible to not have separate field `LeaseStressers`, and put them all in `Stressers`. And just allocate different weights? https://github.com/coreos/etcd/blob/master/tools/functional-tester/etcd-tester/stresser.go#L181-L192